runhouse/servers/http/http_server.py

```diff
@@ -348,6 +349,18 @@ async def install_package(request: Request, params: InstallPackageParams):
                 e, traceback.format_exc(), from_http_server=True
             )
 
+    @staticmethod
+    @app.post("/run_bash")
+    @validate_cluster_access
+    async def run_bash(request: Request, params: RunBashParams):
+        try:
+            retcodes = await obj_store.arun_bash_command_on_all_nodes(params.command)
```

Comment:
We previously had a `require_outputs` option to either only return retcodes or to also include outputs. Can we preserve that here? I think it's pretty important